charts/osdfir-infrastructure/README.md

```diff
@@ -240,6 +241,27 @@ kubectl delete pvc -l release=osdfir-release
 | `turbinia.redis.replica.resources.limits`   | The resources limits for the Redis replica containers                                        | `{}`             |
 | `turbinia.redis.replica.resources.requests` | The requested resources for the Redis replica containers                                     | `{}`             |
 
+### Oauth2 Proxy configuration parameters
+
+| Name                                                                        | Description                                                                            | Value                         |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- | ----------------------------- |
+| `turbinia.oauth2proxy.enabled`                                              | Enables the Oauth2 Proxy deployment                                                    | `false`                       |
+| `turbinia.oauth2proxy.service.type`                                         | OAuth2 Proxy service type                                                              | `ClusterIP`                   |
+| `turbinia.oauth2proxy.service.port`                                         | OAuth2 Proxy service HTTP port                                                         | `8080`                        |
+| `turbinia.oauth2proxy.service.annotations`                                  | Additional custom annotations for OAuth2 Proxy service                                 | `{}`                          |
+| `turbinia.oauth2proxy.configuration.turbiniaSvcPort`                        | Turbinia service port referenced from `.Values.service.port` to be used in Oauth setup | `8000`                        |
```

Comment:
sentence ends with some words missing

Reply:
Double checked `values.yaml` and it matches description there. Maybe sentence just sounds off?

Reply:
Nope, read it again and it makes sense.